buildkite/test_plans.go

```diff
@@ -0,0 +1,50 @@
+package buildkite
+
+import "fmt"
+
+type TestPlansService struct {
+	client *Client
+}
+
+type Case struct {
+	Path              *string `json:"path,omitempty" yaml:"path,omitempty"`
+	EstimatedDuration *int `json:"estimated_duration,omitempty" yaml:"estimated_duration,omitempty"`
+}
+
+type Tests struct {
+	Format *string `json:"format,omitempty" yaml:"format,omitempty"`
+	Cases  []Case  `json:"cases,omitempty" yaml:"cases,omitempty"`
+}
```

Comment:
Everything else has a singular, would we have a Test struct or is that just overkill? Is there ever a time when we would need a singular test? Or is `test == case`?

Reply:
Roughly speaking `tests == case` is true. At the moment, an individual case is a file containing tests that is run by the test splitter. The `format` is set to `file` to denote that all cases within the cases array are file-level.